.eslintrc.js

```diff
@@ -42,7 +48,7 @@ module.exports = {
     {
       files: ['**/*.ts?(x)'],
       parserOptions: {
-        project: ['./tsconfig.json'],
+        project: ['./tsconfig.lint.json'],
```

Comment:
I had to generate different TS config files depending on their purpose to avoid clashing types between Cypress and Jest. You can read more about it here: https://docs.cypress.io/guides/tooling/typescript-support#Clashing-types-with-Jest